### DISCUSSION TITLE
Is there any support for zoning ?

### DISCUSSION THREAD

**Discussion by madbrain76** [dahlb/ha_carrier#109] [General]
I have 2 infinity systems, one with 6 zones and another with 4 zones. That's 2 master thermostats, and 8 slave thermostats.

Is zoning supported by this integration at all ? What about multiple systems?

Currently, I have not purchased the Carrier thermostat that have IP connectivity yet. I would be happy to be the guinea pig if the support is currently missing but could be added with additional testing.

**Comment by Avpman2** [comment]
+1

**Comment by ashman5555** [comment]
I have a Bryant system with the Evolution Connex system control and two zones. Both are represented and controllable in Home Assistant and HomeKit.

**Comment by soundneedle** [comment]
I know this is old AF, but I have 2 systems---one with 3 zones the other with 2 zones. This HA integration works with all of it.

**Reply by ekobres** [reply]
Yes. Zones work.

**Reply by madbrain76** [reply]
Glad to hear. I ended up sticking with old Carrier infinity SYSTXCCUIZ01-V, which lack any kind of Ethernet, Wifi or IP connectivity. I installed two Raspberry Pi 3B+, each with an RS-485 interface, and connected them to the ABCD bus of each furnace.  I then installed Infinitive on those PIs, and use it with the MQTT integration. This provides a fully local solution in Home assistant. It will not die when Carrier eventually shuts down the cloud server for their API. I am also able to keep my thermostat that has proper physical controls, not a hard to read touch screen display with inaccessible color scheme with my macular degeneration. Saving a few thousand dollars in new thermostats was priceless, too.

**Reply by ekobres** [reply]
Yeah, if they ever lock down the API that's a nice option. It actually could be a fun project to make a headless version that works seamlessly as an HA integration and add some of the missing features. Definitely a lot cleaner and simpler to plug into the ABCD than to emulate Carrier's cloud, which is what infinitude was doing - plus you get the benefit of all of Carrier's firmware updates and their app.

At the moment this integration offers great functionality for those of us that already have the connected smart thermostats.
